src/Raven.Server/Commercial/LicenseStatus.cs

```diff
@@ -86,21 +91,37 @@ public LicenseType Type
             }
         }
 
-        public int Version => GetValue<int?>("version") ?? -1;
+        public Version Version
```

Comment:
Let's revert this change. This is valid only in 6.0.

Reply:
Done